src/main.rs

```diff
@@ -75,11 +78,13 @@ To render a local file (for testing):
     $ tldr --render /path/to/file.md
 ";
 const ARCHIVE_URL: &str = "https://github.com/tldr-pages/tldr/archive/master.tar.gz";
+const PAGER_COMMAND: &str = "less -r";
```

Comment:
I'd switch to `-R` by default, so that only color escape codes are interpreted by default. It's safer that way.

```suggestion
const PAGER_COMMAND: &str = "less -R";
```